core/bin/zksync_tee_prover/src/api_client.rs

```diff
@@ -91,6 +91,9 @@ impl TeeApiClient {
         root_hash: H256,
         tee_type: TeeType,
     ) -> Result<(), TeeProverError> {
+        if tee_type == TeeType::None {
+            tracing::info!("Not submitting proof from none TEE.");
+        }
```

Comment:
Just to be clear: It's intentional that the proof is still submitted below via API, right?

Reply:
oops.. good catch... thanks! Fixed